grid-demo/package.json

```diff
@@ -22,6 +22,7 @@
     "@clr/angular": "~13.5.0",
     "@clr/ui": "~13.5.0",
     "@faker-js/faker": "^7.3.0",
+    "moment": "2.29.4",
```

Comment:
moment is dead, is there nothing else we can use?

Reply:
agreed. I'll switch to date-fns